frontend/package.json

```diff
@@ -13,6 +13,7 @@
     "@types/react": "^16.14.2",
     "@types/react-bootstrap": "^0.32.24",
     "@types/react-dom": "^16.9.10",
+    "chokidar": "^3.5.2",
```

Comment:
Was this intentional?

Reply:
It wasn't building for me locally unless I added that dependency. :man_shrugging: